### DISCUSSION TITLE
PR required reviewer not getting added

### DISCUSSION THREAD

**Discussion by marunmohan** [renovatebot/renovate#23295] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.4.0

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I am trying to add required reviewer for the PR (Azure devops). But it is not getting added for the PR's. Below is the renovate.json config and I have attached the logs

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":prHourlyLimitNone",
    ":prConcurrentLimit20"
  ],
  "branchPrefix": "renovate-",
  "reviewers": [
    "required: test@email.com"
  ],
  "flux": {
    "fileMatch": ["\\.yaml$"]
  },
  "kubernetes": {
    "fileMatch": ["\\.yaml$"],
     "ignorePaths": ["base/infrastructure/crds/istio/","base/infrastructure/components/istio/deploy/"]
  },
  "helm-values": {
    "fileMatch": ["\\.yaml$"]
[log1.txt](https://github.com/renovatebot/renovate/files/12013353/log1.txt)

  },
  "bicep": {
    "fileMatch": ["\\.bicep$"]
  },
  "helmv3": {
    "fileMatch": ["\\Chart.yaml$"]
  },
  "packageRules": [
    {
      "description": "Disable docker image update of flux and istio",
      "matchPackagePatterns": [
        "^ghcr.io/fluxcd/.*",
        "^docker.io/istio/.*" 
      ],
      "enabled": false
    },
    {
      "packagePatterns": ["^ghcr.io/kedacore/"],
      "groupName": "Keda"
    },
    {
      "groupName": "bicep-files",
      "matchFiles": ["**/*.bicep"]
    }    
  ]
}
```	

[log1.txt](https://github.com/renovatebot/renovate/files/12013356/log1.txt)

### What you tried so far.

nope

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
The example in the docs (https://docs.renovatebot.com/configuration-options/#reviewers) has not space after `:`:

```
{
  "reviewers": ["required:bar"]
}
```

**Reply by marunmohan** [reply]
I tried without space but still no effect.

**Reply by rarkins** [reply]
Reviewers are only added at PR creation time, please capture a log of a _new_ PR being created with this config
